config/notify.yml

```diff
@@ -0,0 +1,2 @@
+api_key: <%= ENV['GOVUK_NOTIFY_API_KEY'] || "TEST_KEY"%>
+template_id: <%= ENV['GOV_UK_NOTIFY_TEMPLATE_ID'] || "80c6e214-8ed3-4437-bf57-79005ec74719" %>
```

Comment:
Can we stick with either `GOVUK` or `GOV_UK` for environment variable names?

Reply:
`GOVUK` is the one that's consistently used in other apps

Reply:
fixed in 1cf52acf8c2c181b83eff5668df537ff1be6d5ce